doajtest/unit/test_ingestarticles.py

```diff
@@ -103,6 +103,7 @@ def setUp(self):
         self.ftp = ftplib.FTP
 
         self.upload_dir = app.config["UPLOAD_DIR"]
+        self.ingest_articles_retries = app.config.get("HUEY_TASKS").get("ingest_articles").get("retries")
```

Comment:
`AttributeError: 'NoneType' object has no attribute 'get'` if HUEY_TASKS is not defined.

Since this is a test, I would suggest just using `app.config['HUEY_TASKS']['ingest_articles']['retries']`. The test doesn't seem to make sense without this value being defined.